clang/test/CodeGen/aarch64-sve-intrinsics/acle_sve_sel.c

```diff
@@ -63,6 +63,20 @@ svint32_t test_svsel_s32(svbool_t pg, svint32_t op1, svint32_t op2)
   return SVE_ACLE_FUNC(svsel,_s32,,)(pg, op1, op2);
 }
 
+// CHECK-LABEL: @test_svsel_s32_ptrue(
```

Comment:
I'm not sure if this test really adds any more value, since the other test in Transforms/InstCombine/AArch64/sve-intrinsic-sel.ll is also testing the InstCombine.

Reply:
remove the test `test_svsel_s32_ptrue`, thanks @david-arm